app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<span class="user_profile"><h2><%=@user.name.capitalize%>'s Profile</h2></span>
+
+<div id="friends">
+
+  <% @user.user_friends.each do |friend| %>
+    <div class="friends-list">
+      <%= friend.find_friend.name %>
```

Comment:
This is super confusing, because of your naming. This line is actually doing this:
1) `friend` is a _Friend_ object, which represents a _friendship_, not a friend
2) `find_friend` is not accessing a `Friend`, but is rather calling `User` ! Check it out [here](https://github.com/nyc-foxes-2017/BeMyFriend/blob/2bf466874b65061463e70ff6b1452b323187a0e2/app/models/friend.rb#L29-L31).
3) Thus `name` is being called on a `User`

This is what happens when naming is not accurate.